doc/source/user_guide/style.ipynb

```diff
@@ -9,9 +9,183 @@
+    "[formatfuncindex]: ../reference/api/pandas.io.formats.style.Styler.format_index.rst\n",
+    "[relabelfunc]: ../reference/api/pandas.io.formats.style.Styler.relabel_index.rst\n",
+    "[hidefunc]: ../reference/api/pandas.io.formats.style.Styler.hide.rst\n",
+    "[concatfunc]: ../reference/api/pandas.io.formats.style.Styler.concat.rst"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Formatting the Display\n",
+    "\n",
+    "### Formatting Values\n",
+    "\n",
+    "The [Styler][styler] distinguishs the *display* value from the *actual* value, in both datavalues and index or columns headers. To control the display value, the text is printed in each cell as string, and we can use the [.format()][formatfunc] and [.format_index()][formatfuncindex] methods to manipulate this according to a [format spec string][format] or a callable that takes a single value and returns a string. It is possible to define this for the whole table, or index, or for individual columns, or MultiIndex levels. We can also overwrite index names\n",
+    "\n",
+    "Additionally, the format function has a **precision** argument to specifically help formatting floats, as well as **decimal** and **thousands** separators to support other locales, an **na_rep** argument to display missing data, and an **escape** and **hyperlinks** arguments to help displaying safe-HTML or safe-LaTeX. The default formatter is configured to adopt pandas' global options such as `styler.format.precision` option, controllable using `with pd.option_context('format.precision', 2):` \n",
+    "\n",
+    "[styler]: ../reference/api/pandas.io.formats.style.Styler.rst\n",
+    "[format]: https://docs.python.org/3/library/string.html#format-specification-mini-language\n",
+    "[formatfunc]: ../reference/api/pandas.io.formats.style.Styler.format.rst\n",
+    "[formatfuncindex]: ../reference/api/pandas.io.formats.style.Styler.format_index.rst\n",
+    "[relabelfunc]: ../reference/api/pandas.io.formats.style.Styler.relabel_index.rst"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "df = pd.DataFrame({\n",
+    "    \"strings\": [\"Adam\", \"Mike\"],\n",
+    "    \"ints\": [1, 3],\n",
+    "    \"floats\": [1.123, 1000.23]\n",
+    "})\n",
+    "df.style \\\n",
+    "  .format(precision=3, thousands=\".\", decimal=\",\") \\\n",
+    "  .format_index(str.upper, axis=1) \\\n",
+    "  .relabel_index([\"row 1\", \"row 2\"], axis=0)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Using Styler to manipulate the display is a useful feature because maintaining the indexing and datavalues for other purposes gives greater control. You do not have to overwrite your DataFrame to display it how you like. Here is a more comprehensive example of using the formatting functions whilst still relying on the underlying data for indexing and calculations."
```

Comment:
```suggestion
    "Using Styler to manipulate the display is a useful feature because maintaining the indexing and data values for other purposes gives greater control. You do not have to overwrite your DataFrame to display it how you like. Here is a more comprehensive example of using the formatting functions whilst still relying on the underlying data for indexing and calculations."
```